tachyon/zk/base/halo2/halo2_prover_test.h

```diff
@@ -22,6 +22,7 @@ class Halo2ProverTest : public testing::Test {
   constexpr static size_t kDomainSize = kMaxDegree + 1;
   constexpr static size_t kMaxExtendedDegree = (size_t{1} << 7) - 1;
   constexpr static size_t kExtendedDomainSize = kMaxExtendedDegree + 1;
+  constexpr static size_t kQuotientPolyDegree = 3;
```

Comment:
If this value is the degree of the `ConstraintSystem`, 
it should be obtained through `ConstraintSystem.ComputeDegree()`.